service/frontend/redirection_interceptor.go

```diff
@@ -177,8 +177,9 @@ func (i *RedirectionInterceptor) Intercept(
 		return i.handleRedirectAPIInvocation(ctx, req, info, handler, methodName, respAllocFn, namespaceName)
 	}
 
-	// this should not happen, log warn and move on
-	i.logger.Warn(fmt.Sprintf("RedirectionInterceptor encountered unknown API: %v", methodName))
+	// this should not happen for frontend APIs but ok for admin and other future handlers
+	// log warn and move on
+	i.logger.Debug(fmt.Sprintf("RedirectionInterceptor encountered unknown API: %v", methodName))
```

Comment:
```suggestion
	i.logger.Debug("RedirectionInterceptor encountered unknown API", tag.Name(info.FullMethod))
```
I'd rather see the full name to know what service it's in